### ISSUE TITLE
Gauge LSP - run test cases

### ISSUE THREAD

**Issue by sguptatw** [getgauge/gauge#832]
**Proposed behavior**
Option to run Data driven specification

**Out of scope**
- Debugging
- Running a particular table row

**Steps**
* Provide a Code lens link to run all the table rows of the specification in parallel 
* Provide a link to run specification against all table rows sequentially.

## Acceptance criteria
- [x] Execute all the scenarios in the specification, show the results in the output window.
- [x] Execute all the test cases in the specification in parallel

## Test cases
- [x] Should be able to run all test cases sequentially
- [x] Should be able to run all test cases in parallel
- [x] Should be able to run all scenarios in the specification
- [x] The results should be displayed in the console

## Issues
> The results should be displayed in the console
- [ ] Deprecated messages are shown in IDE console output  https://github.com/getgauge/gauge-vscode/issues/13
- [x] Link to html-report is not working as expected - https://github.com/getgauge/gauge-vscode/issues/14

**Comment by BugDiver** [comment]
Should be available in gauge nightly >= 05-10-2017 and gauge-vscode nightly >= 6-10-2017
